scripts/run-travis-tests.sh

```diff
@@ -40,8 +40,10 @@ PYTHONPATH=. run runtests ./scripts/runtests.py 2>&1
 PYTHONPATH=. run pytest pytest -sv tests
 PYTHONPATH=. run canonicalizer_tests pytest testcases/elichika_tests/canonicalizer
 
-PYTHONPATH=. run train_mnist python3 examples/mnist/train_mnist.py \
-     -d native --compile -I 3 --use-fake-data
+PYTHONPATH=. run train_mnist_export python3 examples/mnist/train_mnist.py \
+     -d -1 --export /tmp/tmp_mnist_model.onnx -I 3 --use-fake-data
```

Comment:
Is `-d -1` intentional? s/-1/native/?

Reply:
I changed the translator for MNIST example to onnx-chainer, and `native` did not work for onnx-chainer.